circle.yml

```diff
@@ -1,11 +0,0 @@
-machine:
-  ruby:
-    version: 2.2.3
-
-dependencies:
-  pre:
-    - curl -fsSL https://git.io/v2Ifn | bash
-
-database:
-  override:
-    - ':' # no-op
```

Comment:
@nate00 What does this line do?

I noticed a bunch of junk in the output when running the tests. Maybe it's related to the removal?

```
W, [2017-09-24T21:52:27.210917 #210]  WARN -- : localhost:11211 failed (count: 0) Errno::EADDRNOTAVAIL: Cannot assign requested address - connect(2) for "localhost" port 11211
E, [2017-09-24T21:52:27.221364 #210] ERROR -- : DalliError: localhost:11211 is down:  . If you are sure it is running, ensure memcached version is > 1.4.
W, [2017-09-24T21:52:27.252599 #210]  WARN -- : localhost:11211 failed (count: 1) Errno::EADDRNOTAVAIL: Cannot assign requested address - connect(2) for "localhost" port 11211
W, [2017-09-24T21:52:27.252783 #210]  WARN -- : localhost:11211 is down
E, [2017-09-24T21:52:27.252935 #210] ERROR -- : DalliError: No server available
E, [2017-09-24T21:52:27.253243 #210] ERROR -- : DalliError: No server available
E, [2017-09-24T21:52:27.253501 #210] ERROR -- : DalliError: No server available
E, [2017-09-24T21:52:27.255835 #210] ERROR -- : DalliError: No server available
```